hub/src/main/java/com/hub/HubController.java

```diff
@@ -1,30 +1,46 @@
 package com.hub;
 
-import org.springframework.http.ResponseEntity;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
 import org.springframework.web.bind.annotation.*;
 
-import java.io.OutputStream;
+import java.io.*;
 
 import com.google.gson.Gson;
 import com.google.gson.JsonObject;
 import com.google.gson.JsonParser;
 
 import javax.annotation.PostConstruct;
 import java.net.HttpURLConnection;
-import java.net.MalformedURLException;
+import java.net.URI;
 import java.net.URL;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
 import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.util.HashMap;
 import java.util.List;
 
+
 @RestController
+@EnableScheduling
 public class HubController {
 
     private final ServerHub hub;
     private final HubRepository repository;
+    private String leader;
```

Comment:
any reason the `leader` is a String here and an int in the book stores?

Reply:
It was a pain to send as a Long so we kept it as a String for sending purposes. It was like 40 minutes of trying to find work arounds so we eventually just stuck to Strings which we know how to work

Reply:
gotcha!